hooks/tk-multi-workfiles2/scene_operation_tk-nuke.py

```diff
@@ -353,7 +363,11 @@ def _scene_operation_hiero_nukestudio(
             hiero.core.events.sendEvent("kBeforeProjectLoad", None)
 
             # open the specified script
-            hiero.core.openProject(file_path.replace(os.path.sep, "/"))
+            project = hiero.core.openProject(file_path.replace(os.path.sep, "/"))
+
+            # store project context in tag.note()
+            context_str = engine_context.serialize(engine_context)
+            self.add_context_to_project(project, context_str)
```

Comment:
We should also do this at new project creation too, since the context won't change even after the user saves the project.

Reply:
addressed the comments .